.github/workflows/e2e-component-tests-embedding-sdk-cross-version.yml

```diff
@@ -128,8 +128,8 @@ jobs:
       DISPLAY: ""
       # Any env starting with `CYPRESS_` will be available to all Cypress tests via `Cypress.env()`
       # Example: you can get `CYPRESS_FOO` with `Cypress.env("FOO")`
-      CYPRESS_ALL_FEATURES_TOKEN: ${{ secrets.ENTERPRISE_TOKEN }}
-      CYPRESS_NO_FEATURES_TOKEN: ${{ secrets.E2E_STARTER_TOKEN }}
+      CYPRESS_ALL_FEATURES_TOKEN: ${{ secrets.MB_PRO_CLOUD_TOKEN }}
+      CYPRESS_NO_FEATURES_TOKEN: ${{ secrets.MB_STARTER_CLOUD_TOKEN }}
```

Comment:
I really wonder if all our tests will pass with the cloud features on 🤔

Reply:
Let's let it run and see what happens!

Reply:
Cool.
This failed.

Wanna try the self-hosted now and we can compare the failures?

Reply:
https://github.com/metabase/metabase/pull/59754/commits/84708de456b50c5e66bf38fb0e8049e0f06a433b